decide/utils.cpp

```diff
@@ -0,0 +1,63 @@
+#include <decide/utils.hpp>
+
+double angle_between_points(Coordinate A, Coordinate B, Coordinate C) {
```

Comment:
Looks correct. Would be nice if we could clearly specify (in a comment right above, to document the function) which argument is the vertex of the angle.

Reply:
Specify this either here or in the .hpp file or both, not sure.

Reply:
There's documentation in the header-file. I  don't know whats the correct way to do it but VS Code seems to be able to pick it up.